public/app/features/admin/ActionButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC, useContext } from 'react';
+import { css, cx } from 'emotion';
+import { ThemeContext, stylesFactory } from '@grafana/ui';
+import { GrafanaTheme } from '@grafana/data';
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  return {
+    button: css`
+      text-decoration: underline;
+      color: ${theme.colors.blue95};
+    `,
+  };
+});
+
+interface Props {
+  text: string;
+  className?: string;
+  onClick(): void;
+}
+
+export const ActionButton: FC<Props> = (props: Props) => {
+  const { onClick, text, className } = props;
+  const theme = useContext(ThemeContext);
```

Comment:
you don't need to use `useContext`-  there is `useTheme` hook that does it for you.

Reply:
Cool, will do.